Firestore/core/src/firebase/firestore/remote/exponential_backoff.h

```diff
@@ -26,6 +26,8 @@ namespace firebase {
 namespace firestore {
 namespace remote {
 
+namespace chr = std::chrono;
```

Comment:
Style guide [frowns](https://google.github.io/styleguide/cppguide.html#Namespaces) upon namespace aliases in header files: _"Do not use Namespace aliases at namespace scope in header files except in explicitly marked internal-only namespaces, because anything imported into a namespace in a header file becomes part of the public API exported by that file."_. I'd suggest moving this to the corresponding source file.

Optionally, you may introduce an alias for the unwieldy type of `last_attempt_time_`, similar to `Milliseconds`:
```cpp
using Milliseconds = util::AsyncQueue::Milliseconds;
using TimePoint = std::chrono::time_point<std::chrono::steady_clock>;
// ...
TimePoint last_attempt_time_;
```